ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
Changing the backend can actually be done from the invoker. Python logging handlers are essentially that (e.g. adding a file handler and disabling a stream handler can be done after import before everything runs). Enforcing consistency might be necessary one day, but for now I like the simplicity of calling 'log = logging.getLogger(__name__)'